### ISSUE TITLE
Document what are the names for integer vectors.

### ISSUE THREAD

**Issue by iMplode-nZ** [termhn/ultraviolet#96]
These don't show up in the documentation and there is nowhere stating whether it's "Vec3I" or "Vec3D" or "Vec3U" or what.

**Comment by termhn** [comment]
Vec3I = integer (i32)
Vec3U = unsigned integer (u32)
Vec3D = double (f64)

It's unfortunate that these don't show up in the documentation, I'll have to think on what the best way to document this in a more visible spot is

**Comment by iMplode-nZ** [comment]
Yeah, I figured that out... by looking at the source. Also, you're wrong, it's IVec3

**Comment by termhn** [comment]
No need to be so aggressive about it and thanks for the intentionally misleading naming in the initial post :/

**Comment by iMplode-nZ** [comment]
(Well I didn't actually figure out until after I made this post)

**Comment by thenlevy** [comment]
> It's unfortunate that these don't show up in the documentation, I'll have to think on what the best way to document this in a more visible spot is

I suggest to document this in the `feature` section at the root of the crate, since that's where the f64 types are documented.

Also I believe you can add
```
[package.metadata.docs.rs]

# Features to pass to Cargo (default: [])
features = ["f64", "int"]
```
to the Cargo.toml and the types will be documented on doc.rs

**Comment by termhn** [comment]
Fixed in e5bc083
